tests/pulp_test.py

```diff
@@ -1,37 +1,40 @@
 import json
-
 import pytest
 
-
 PULP_HOST = 'localhost'
-PULP_PORT = 8080
-
+PULP_API_PORT = 24817
+PULP_CONTENT_PORT = 24816
 
 @pytest.fixture(scope="module")
 def pulp_status_curl(host):
-    return host.run(f"curl -k -s -w '%{{stderr}}%{{http_code}}' http://{PULP_HOST}:{PULP_PORT}/pulp/api/v3/status/")
-
+    return host.run(f"curl -k -s -w '%{{stderr}}%{{http_code}}' http://{PULP_HOST}:{PULP_API_PORT}/pulp/api/v3/status/")
 
 @pytest.fixture(scope="module")
 def pulp_status(pulp_status_curl):
     return json.loads(pulp_status_curl.stdout)
 
+def test_pulp_api_service(host):
+    pulp_api = host.service("pulp-api")
+    assert pulp_api.is_running
+    assert pulp_api.is_enabled
 
-def test_pulp_service(host):
-    pulp = host.service("pulp")
-    assert pulp.is_running
-    assert pulp.is_enabled
+def test_pulp_content_service(host):
+    pulp_content = host.service("pulp-content")
+    assert pulp_content.is_running
+    assert pulp_content.is_enabled
 
+def test_pulp_worker_service(host):
+    pulp_worker = host.service("pulp-worker")
+    assert pulp_worker.is_running
+    assert pulp_worker.is_enabled
 
-def test_pulp_port(host):
-    pulp = host.addr(PULP_HOST)
-    assert pulp.port(PULP_PORT).is_reachable
-
-
-def test_pulp_status(pulp_status_curl):
-    assert pulp_status_curl.succeeded
-    assert pulp_status_curl.stderr == '200'
```

Comment:
It would be nice to keep this check in to verify Pulp actually thinks it's healthy.

Reply:
Definitely! I've reintroduced this.